src/audio/google/google_rtc_audio_processing.c

```diff
@@ -441,6 +454,9 @@ static struct comp_dev *google_rtc_audio_processing_create(
 		goto fail;
 	bzero(cd->output_buffer, cd->num_frames * sizeof(cd->output_buffer[0]));
 	cd->output_buffer_frame_index = 0;
+	for (channel = 0; channel < cd->num_capture_channels; channel++)
+		cd->aec_reference_buffer_ptrs[channel] =
+			&cd->aec_reference_buffer[channel * cd->num_frames];
```

Comment:
Confused: this is looping over the same array as the num_aec_reference_channels loop above and initializing the same indexes to the same values.   Looks like semantically it's identical to looping once over the maximum of the two counts?  Or maybe it's supposed to be initializing a different array?

Reply:
very likely a copy paste error